packages/app/src/lib/client/index.ts

```diff
@@ -143,6 +144,21 @@ export class SunriseStakeClient {
     return this.provider.sendAndConfirm(createATAIx, []);
   }
 
+  createGSolTokenAccountInstruction(): TransactionInstruction {
```

Comment:
Looks good - it was on my TODO list to combine the ATA creation and deposit - would you mind changing deposit() to use this instead of createGSolTokenAccount for consistency(and then you can remove createGSolTokenAccount)?

Reply:
Alright. Will do